.github/workflows/gradle.yml

```
@@ -37,8 +36,6 @@ jobs:
         build-scan-terms-of-use-url: "https://gradle.com/terms-of-service"
         build-scan-terms-of-use-agree: "yes"
         dependency-graph: generate-and-submit
-    - name: Run build with Gradle wrapper
-      id: gradle
-      env:
-        LANGCHAIN4J_CHAT_MODEL_OPENAI_API_KEY: "${{ secrets.LANGCHAIN4J_CHAT_MODEL_OPENAI_API_KEY }}"      
-      run: ./gradlew build
\ No newline at end of file
+    - name: Build batch-rest-repository
+      id: gradle-batch-rest-repository
+      run: ./gradlew batch-rest-repository:build
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)